### ISSUE TITLE
TagBot fails to run with GithubException

### ISSUE THREAD

**Issue by jagot** [JuliaRegistries/TagBot#61]
Please see
https://github.com/jagot/SolverTraces.jl/runs/434958410?check_suite_focus=true#step:3:64

**Comment by longemen3000** [comment]
https://github.com/longemen3000/CachedFunctions.jl/runs/435186500?check_suite_focus=true

**Comment by longemen3000** [comment]
```
 github.GithubException.GithubException: 422 {"message": "The since parameter needs to be in ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ", "documentation_url": "https://developer.github.com/v3/issues/#list-issues-for-a-repository"}
```
happens on tagbot 1.5.3

**Comment by christopher-dG** [comment]
Fixed by 44ace62d13461f7c88afc721b5504c89980c6076 I think. We'll find out soon!

I'm not sure if this is a subtle breaking change in GitHub's API, or if this is just the first time that TagBot has tried to handle a package's first release since moving the Docker image to Alpine. Basically, GitHub doesn't like this timestamp: `0001-01-01T00:00:00Z` (what Alpine gives us) but it likes `1-01-01T00:00:00Z` (what I get on my own machine). Now that the default starting year is 4 digits, the difference shouldn't matter anymore.
